projects/packages/forms/src/dashboard/index.tsx

```diff
@@ -49,14 +49,10 @@ window.addEventListener( 'load', () => {
 					path: 'responses',
 					element: <Inbox />,
 				},
-				...( config( 'enableIntegrationsTab' )
-					? [
-							{
-								path: 'integrations',
-								element: <Integrations />,
-							},
-					  ]
-					: [] ),
```

Comment:
Note: This change means that the /integrations path will always be present on the forms dashboard. But if the integrations flag is off, we will not show the Integrations TAB/navigation, so the only way to get to this screen is to know and enter the path directly. 

If we want to go further, I'd say that on the integrations screen, if integrations are turned off, we display a message that integrations are disabled for this site. But I don't personally think that's needed.